src/main/scala/io/citrine/lolo/validation/Merit.scala

```diff
@@ -0,0 +1,215 @@
+package io.citrine.lolo.validation
+
+import java.util
+
+import io.citrine.lolo.PredictionResult
+import org.knowm.xchart.XYChart
+
+import scala.collection.JavaConverters._
+import scala.util.Random
+
+/**
+  * Real-valued metric on predictions of type T
+  */
+trait Merit[T] {
+
+  /**
+    * Apply the metric to a prediction result and set of ground-truth values
+    *
+    * @return the value of the metric
+    */
+  def evaluate(predictionResult: PredictionResult[T], actual: Seq[T]): Double
+
+  /**
+    * Estimate the metric and the uncertainty in the metric over batches of predicted and ground-truth values
+    *
+    * @param pva predicted-vs-actual data as an iterable over [[PredictionResult]] and ground-truth tuples
+    * @return the estimate of the metric value and the uncertainty in that estimate
+    */
+  def estimate(pva: Iterable[(PredictionResult[T], Seq[T])]): (Double, Double) = {
+    val samples = pva.map { case (prediction, actual) => evaluate(prediction, actual) }
+    val mean: Double = samples.sum / samples.size
+    val variance: Double = (samples.size / (samples.size - 1)) * samples.map(x => Math.pow(x - mean, 2)).sum / samples.size
+    (mean, Math.sqrt(variance / samples.size))
+  }
+}
+
+/**
+  * SSIA
```

Comment:
I think even a simple description would be helpful here. 

```suggestion
  * Square root of the mean square error. For an unbiased estimator, this is equal to the standard deviation of the difference between predicted and actual values.
```

(If I understand this figure of merit correctly, but you probably have a better explanation.)